### PR TITLE
Button-components-changes-https://becurious.edcast.eu/journey/aem-demo-website/cards/109701819

### PR DESCRIPTION
**Pull request by ashankarprakash**

### CODE REVIEW

File: ui.content/src/main/content/jcr_root/conf/aemaacsdemowebsite/settings/wcm/policies/.content.xml [hunk 1]
```diff
@@ -27,6 +27,75 @@
                     <jcr:content jcr:primaryType="nt:unstructured"/>
                 </policy_xfm>
             </languagenavigation>
+            <button jcr:primaryType="nt:unstructured">
+                <policy_1680682147226
+                    jcr:lastModified="{Date}2023-04-13T06:09:02.574-04:00"
```
Comment (manpkaur9) [inline ui.content/src/main/content/jcr_root/conf/aemaacsdemowebsite/settings/wcm/policies/.content.xml:32]: Please remove these fields, same comment as last PR

File: ui.frontend/src/main/webpack/site/_variables.scss [hunk 2]
```diff
@@ -16,3 +16,15 @@ $color-link:            #2020E0;
...
+ 
+$color-red: red;
+$color-blue: blue;
+$color-white: white;
+$button-radius: 7px;
+$button-padding: 10px 20px;
+$button-font: bold;
+$color-white: white;
+$button-alignment-inherit: inherit;
+$button-alignment-left: left;
```
Comment (manpkaur9) [inline ui.frontend/src/main/webpack/site/_variables.scss:28]: unnecessary variables for inherit, left, right and center

**Review by manpkaur9 — CHANGES_REQUESTED** [review]
image


The styles are not working as expected

**Review by manpkaur9 — CHANGES_REQUESTED** [review]
image

Also, there is this difference between the two buttons

File: ui.content/src/main/content/jcr_root/conf/aemaacsdemowebsite/settings/wcm/policies/.content.xml [hunk 3]
```diff
@@ -27,6 +27,75 @@
                     <jcr:content jcr:primaryType="nt:unstructured"/>
                 </policy_xfm>
             </languagenavigation>
+            <button jcr:primaryType="nt:unstructured">
```
[the remark this answers is not in the document]
Comment (ashankarprakash) [reply ui.content/src/main/content/jcr_root/conf/aemaacsdemowebsite/settings/wcm/policies/.content.xml:30]: Added in the latest commit.

File: ui.content/src/main/content/jcr_root/conf/aemaacsdemowebsite/settings/wcm/policies/.content.xml (hunk 1 above)
Comment (ashankarprakash) [reply ui.content/src/main/content/jcr_root/conf/aemaacsdemowebsite/settings/wcm/policies/.content.xml:32]: Removed in the latest commit.

File: ui.frontend/src/main/webpack/site/_variables.scss (hunk 2 above)
Comment (ashankarprakash) [reply ui.frontend/src/main/webpack/site/_variables.scss]: Removed in the latest commit.

**Review by manpkaur9 — APPROVED** [review]

**Pull request merged**